<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>LCC-TR</string>
  </property>
  <property name="whatsThis">
   <string>一个在官方没时间维护LLC汉化工具时的一个小型替代品</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;欢迎使用LLC-TemporaryReplacer&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>11</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="lineWidth">
       <number>7</number>
      </property>
      <property name="text">
       <string>注意：这个程序不是LLC官方制作。</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>游戏目录</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="Path">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>999</width>
           <height>27</height>
          </size>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ChoosePathBtn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>35</height>
          </size>
         </property>
         <property name="text">
          <string>浏览…</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="Execute">
       <attribute name="title">
        <string>安装/卸载</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>安装/Install</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QWidget" name="widget_2" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="label_4">
                <property name="font">
                 <font>
                  <family>微软雅黑</family>
                  <pointsize>12</pointsize>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>开始安装/更新</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="InstallBtn">
                <property name="maximumSize">
                 <size>
                  <width>150</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="text">
                 <string>安装</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget_3" native="true">
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QLabel" name="label_5">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="font">
                 <font>
                  <family>微软雅黑</family>
                  <pointsize>10</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>该脚本根据月亮计划提供的多语言接口教程制作，误封可能不是本软件造成的，请谨慎区别！！</string>
                </property>
                <property name="wordWrap">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>卸载/Uninstall</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QWidget" name="widget_4" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="label_6">
                <property name="font">
                 <font>
                  <family>微软雅黑</family>
                  <pointsize>12</pointsize>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>开始卸载</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="UninstallBtn">
                <property name="maximumSize">
                 <size>
                  <width>150</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="text">
                 <string>卸载</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_5" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>进度：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QProgressBar" name="progressBar">
             <property name="value">
              <number>0</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Custom">
       <attribute name="title">
        <string>自定义</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QWidget" name="widget_6" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QFrame" name="frame">
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLabel" name="ChooseFont">
                <property name="font">
                 <font>
                  <family>微软雅黑</family>
                  <pointsize>12</pointsize>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>选择字体</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="ChooseFontBtn_2">
                <property name="maximumSize">
                 <size>
                  <width>150</width>
                  <height>35</height>
                 </size>
                </property>
                <property name="text">
                 <string>Choose/选择</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="frame_7">
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QLabel" name="FontLabel">
                <property name="font">
                 <font>
                  <family>微软雅黑</family>
                  <pointsize>12</pointsize>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>选择的字体：思源黑体</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="ResetBtn">
                <property name="maximumSize">
                 <size>
                  <width>150</width>
                  <height>35</height>
                 </size>
                </property>
                <property name="text">
                 <string>Reset/重置</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_8" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="log">
       <attribute name="title">
        <string>日志</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <widget class="QTextEdit" name="logtext">
          <property name="cursor" stdset="0">
           <cursorShape>ArrowCursor</cursorShape>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Microsoft YaHei UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="About">
       <attribute name="title">
        <string>关于</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>12</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>&lt;span style=&quot;font-size:20pt;&quot;&gt;LLC-TemporaryReplacer&lt;/span&gt;
&lt;br/&gt;
&lt;span style=&quot;font-size:8pt;color:red;&quot;&gt;IM GOING TO MAKE IT HAPPEN EVEN IF NO ONE USES IT!&lt;/span&gt;
&lt;br/&gt;
&lt;span style=&quot;font-size:14pt;&quot;&gt;脚本作者：EveGlowLuna（YangChen114514）&lt;/span&gt;
&lt;br/&gt;
&lt;span style=&quot;font-size:14pt;&quot;&gt;本程序将仅在LLC官方无时间完成且LLC工具无法使用时开始维护，以一个正式版发布为终止。&lt;/span&gt;
&lt;br/&gt;
&lt;span style=&quot;font-size:2pt;&quot;&gt;你以为直播问你的只是说说而已？当然不是。所以为什么不理我？（&lt;/span&gt;
&lt;br/&gt;
&lt;span style=&quot;font-size:10pt;&quot;&gt;以上消息没有任何实质性意义，请不要翻看并深究:D&lt;/span&gt;</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
